bundle/src/test/java/com/adobe/acs/commons/rewriter/impl/StaticReferenceRewriteTransformerFactoryTest.java

```diff
@@ -121,6 +121,58 @@ public void test_with_prefix_and_matching_pattern_and_single_host() throws Excep
         assertEquals("//static.host.com/content/dam/flower.jpg", values.get(1).getValue(0));
     }
 
+    @Test
+    public void test_with_prefix_and_matching_pattern_and_single_host_and_replace_host() throws Exception {
+        MockBundle bundle = new MockBundle(-1);
+        MockComponentContext ctx = new MockComponentContext(bundle);
+        ctx.setProperty("prefixes", new String[] { "/content/dam" });
+        ctx.setProperty("attributes", new String[] { "img:src" });
+        ctx.setProperty("host.pattern", "static.host.com");
+        ctx.setProperty("matchingPatterns", "img:src;(\\/content\\/dam\\/.+?\\.(png|jpg))");
+        ctx.setProperty("replaceHost", "true");
```

Comment:
I don't think it really matters since `PropertiesUtil` handles this case, but this should probably just be a boolean `true` not a string.